### ISSUE TITLE
Add link to release notes for Backdrop installed version on status and available updates reports.

### ISSUE THREAD

**Issue by izmeez** [backdrop/backdrop-issues#6587]
## Description of the need

The status report at `admin/reports/status` and available updates report at `admin/reports/updates` both show which version of Backdrop CMS is installed. It would be helpful to have **a link to the release notes** for the version always available.

This is especially useful before and even after installing a new version. 

The new Backdrop 1.28.0 release notes, like others before it, provide important details on changes and new features. The release notes for this version include information on the new configurable config storage feature. 

A simple link to the release notes will be useful and save a lot of time to quickly check when questions arise.

## Proposed solution

Add a link to the release notes of the installed version of Backdrop Core on both the Status and Available Updates Reports.

**Comment by klonos** [comment]
@izmeez any .api.php file is not participating actively in the code that runs as part of the site - it is meant to be used as examples and it is what our API/docs site "consumes" in order to build the various documentation around the APIs.

You don't need the `hook_requirements()` function itself, rather than an actual implementation of that function (a function that has an actual module name instead of `hook_`). The specific hook implementation that you should be looking to tweak is [`system_requirements()`](https://docs.backdropcms.org/api/backdrop/core%21modules%21system%21system.install/function/system_requirements/1). All `hook_requirements()` implementations go in the respective .install file of each module, so you should be looking at the `core/modules/system/system.install` file specifically.

If you happen to join the office hours, I can help you further with general tips on where/how to find the code that needs tweaking 😉 (too much to be typing it all here - sorry)

**Comment by klonos** [comment]
Yes, I believe that `system_requirements()` in `system.install` within the System module is what you need. The rest of the implementations could serve as examples as you said @izmeez 👍🏼
